webapp/channels/src/components/user_settings/display/render_emoticons_as_emoji/render_emoticons_as_emoji.tsx

```diff
@@ -0,0 +1,167 @@
+                                id='user.settings.display.renderEmoticonsAsEmojiTitle'
+                                defaultMessage='Auto-render emoticons as emojis'
+                            />
+                        </legend>
+                        <div className='radio'>
+                            <label>
+                                <input
+                                    id='renderEmoticonsAsEmojiOn'
+                                    type='radio'
+                                    value='true'
+                                    name='renderEmoticonsAsEmoji'
+                                    checked={renderEmoticonsAsEmojiState === 'true'}
+                                    onChange={handleOnChange}
+                                />
+                                <FormattedMessage
+                                    id='user.settings.display.renderEmoticonsAsEmojiOn'
+                                    defaultMessage='On'
+                                />
+                            </label>
+                            <br/>
+                        </div>
+                        <div className='radio'>
+                            <label>
+                                <input
+                                    id='renderEmoticonsAsEmojiOff'
+                                    type='radio'
+                                    value='false'
+                                    name='renderEmoticonsAsEmoji'
+                                    checked={renderEmoticonsAsEmojiState === 'false'}
+                                    onChange={handleOnChange}
+                                />
+                                <FormattedMessage
+                                    id='user.settings.display.renderEmoticonsAsEmojiOff'
+                                    defaultMessage='Off'
+                                />
+                            </label>
+                            <br/>
+                        </div>
+                        <div className='mt-5'>
+                            <FormattedMessage
+                                id='user.settings.display.renderEmoticonsAsEmojiDesc'
+                                defaultMessage='When enabled, text emoticons in messages will automatically be rendered as emojis (For example :D as 😄)'
+                            />
+                        </div>
+                    </fieldset>,
+                ]}
+                setting='renderEmoticonsAsEmoji'
+                submit={handleSubmit}
+                saving={isSaving}
+                serverError={serverError}
+                updateSection={handleUpdateSection}
+            />
+        );
+    }
+
+    return (
+        <SettingItemMin
+            title={
+                <FormattedMessage
+                    id='user.settings.display.renderEmoticonsAsEmojiTitle'
+                    defaultMessage='Auto-render emoticons as emojis'
```

Comment:
Could you change the wording on this to remove the "Auto-" part? I know it disagrees with the text in the ticket, but I don't think "auto-rendering" is the correct term here
```suggestion
                    defaultMessage='Render emoticons as emojis'
```